src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const basePrice = 40;
```

Comment:
The name `basePrice` could be more descriptive. Consider renaming it to `DAILY_RENTAL_RATE` to explicitly state that this is the rate charged per day.